### PR TITLE
Added help section

### LINKED ISSUE #166: [HTML/CSS/JS] Unified info boxes
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [X] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/meilix

**Actual Behaviour**

Currently, we do not have info boxes on the page.

**Expected Behaviour**

I would like to have a link saying `more information` below these entries:

- email
- distro name
- home page
- search engine
- wallpaper

and above tweaks

Clicking this link
- toggles an information text at the same place. This is usable from phones which do not allow hover.
- changes the text to "less information" and "more information" depending on whether it shows the info text.

### PR DESCRIPTION
**Pull request by meets2tarun**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] There is a corresponding issue for this pull request.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes (provide a screenshot or link for test) 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
This PR provides a description help bar view the need of filling of the option.

#### Changes proposed in this pull request:

- A help bar which pop-up while one click on it.
- It gets closed after clicking on a blank screen.

Fixes #166

### CODE REVIEW

**Comment by meets2tarun** [comment]
@rpotter12 
Please see this and tell whether its would be fine to implement like this:

screenshot 2018-07-02 at 09 52 47

screenshot 2018-07-02 at 09 53 03

**Comment by mariobehling** [comment]
Please follow best practices and provide test site and screenshot for all PRs.

I don't understand why there is a template and you don't fill it in or change the text.

**Comment by mariobehling** [comment]
Please follow best practices and provide test site and screenshot for all PRs.

I don't understand why there is a template and you don't fill it in or change the text.

**Comment by rpotter12** [comment]
@meets2tarun I think this will not look good. I think we should use @niccokunzmann suggestion "question mark sign" this will be more appropriate. And when we click on the box a small notification box or an alert box should open which tell what is it. The changes you have applied is not looking too good for just a notification.

**Comment by rpotter12** [comment]
@meets2tarun I will try to implement what I want to say by today.

**Comment by meets2tarun** [comment]
@rpotter12 

Thanks, we can go by your idea also.

File: templates/index.html [hunk 1]
```diff
@@ -283,8 +283,49 @@
...
-  </style>
 
+body {font-family: Arial, Helvetica, sans-serif;}
+
+/* The Modal (background) */
+.modal {
+    display: none; /* Hidden by default */
+    position: fixed; /* Stay in place */
+    z-index: 1; /* Sit on top */
+    padding-top: 100px; /* Location of the box */
```
Comment (tabesin) [inline templates/index.html]: Why use px = pixels?

File: templates/index.html [hunk 2]
```diff
@@ -283,8 +283,49 @@
...
+    left: 0;
+    top: 0;
+    width: 100%; /* Full width */
+    height: 100%; /* Full height */
+    overflow: auto; /* Enable scroll if needed */
+    background-color: rgb(0,0,0); /* Fallback color */
```
Comment (tabesin) [inline templates/index.html]: why not "black"?

File: templates/index.html [hunk 3]
```diff
@@ -283,8 +283,49 @@
...
+    background-color: rgba(0,0,0,0.4); /* Black w/ opacity */
+}
+
+/* Modal Content */
+.modal-content {
+    background-color: #fefefe;
+    margin: auto;
+    padding: 20px;
```
Comment (tabesin) [inline templates/index.html]: Again why pixels?

**Pull request merged**